server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -61,8 +77,45 @@ public TabletsMutator mutateTablets() {
     return new TabletsMutatorImpl(context);
   }
 
+  private void mutateRootGcCandidates(Consumer<RootGcCandidates> mutator) {
+    String zpath = context.getZooKeeperRoot() + ZROOT_TABLET_GC_CANDIDATES;
+    try {
+      context.getZooReaderWriter().mutate(zpath, new byte[0], ZooUtil.PUBLIC, currVal -> {
+        String currJson = new String(currVal, UTF_8);
+
+        RootGcCandidates rgcc = RootGcCandidates.fromJson(currJson);
+
+        log.debug("Root GC candidates before change : {}", currJson);
+
+        mutator.accept(rgcc);
+
+        String newJson = rgcc.toJson();
+
+        log.debug("Root GC candidates after change  : {}", newJson);
+
+        if (newJson.length() > 1 << 18) {
```

Comment:
Could just hard-code something like `256_000`

Reply:
I updated it to a constant.